app/assets/javascripts/service-worker.js.erb

```diff
@@ -258,11 +240,6 @@ self.addEventListener('notificationclick', function(event) {
   }
 });
 
-self.addEventListener('message', function(event) {
-  if('lastAction' in event.data){
-    lastAction = event.data.lastAction;
-  }
-});
```

Comment:
Why are these lines getting deleted?

Reply:
not used anymore.

Reply:
oh, wait, never mind, I get it. The only purpose of this `message` handler was to update the `lastAction` date for `isIdle()`.

You'd probably want to remove the code that fires these events, too.

https://github.com/discourse/discourse/blob/e8509ddf505817a65b8c1f15282a08e4b20cbecc/app/assets/javascripts/discourse/app/lib/push-notifications.js#L26

Reply:
will do